src/App.tsx

```diff
@@ -1,18 +1,91 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useContext, useMemo, useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
```

Comment:
add index.ts file to your TodoList folder, then export your todo list, and simplify your import here

`components/TodosList/index.ts`
```
export * from './TodoList';
```

`App.tsx`
```
import { TodoList } from './components/TodoList'
```